lab_6_pipeline/pipeline.py

```diff
@@ -112,12 +210,21 @@ def convert_morphological_tags(self, tags: str) -> str:  # type: ignore
         """
         Converts the Mystem tags into the UD format
         """
+        tags_list = re.split(r'\W+', re.match(r'[^|]+|', tags).group())
+        ud_tags = []
+        if tags_list[0] in ['PART', 'PR', 'CONJ', 'INTJ', 'COM', 'ADV', 'ADVPRO']:
+            return '_'
+        for tag in tags_list[1:]:
+            for category in self._tag_mapping.keys():
+                if tag in self._tag_mapping[category].keys():
+                    ud_tags.append(f'{category}={self._tag_mapping[category][tag]}')
+        return '|'.join(sorted(ud_tags))
 
     def convert_pos(self, tags: str) -> str:  # type: ignore
         """
         Extracts and converts the POS from the Mystem tags into the UD format
         """
-
+        return self._tag_mapping[self.pos].get(tags)
```

Comment:
you get a fully qualified string with POS and morph tags here. how can you get a value from dict?